src/bin/gpfusion/gpbridgeapi.c

```diff
@@ -273,6 +277,26 @@ void gpbridge_export_start(PG_FUNCTION_ARGS)
 
 }
 
+void parse_namespace(char *uri, char** extracted_namespace){
+	int i = 0;
+	int len = strlen(uri);
+
+	while(uri[i] != '*' && i < len){
```

Comment:
`*` can be part of the PXF location (URL), because we support regex matching for file names (e.g. `file*.txt`).
If this is going to be a permanent part of the URL (not just for secured env), perhaps we can scan from the end? 
Another option is to replace the delimiter.